src/test/suite/fixtures/recorded/selectionTypes/bringWhaleToBeforeFile.yml

```diff
@@ -0,0 +1,37 @@
+languageId: plaintext
```

Comment:
as above, identical to `src/test/suite/fixtures/recorded/positions/bringWhaleToBeforeFile.yml`?